proposals/market_perp_btcusd.go

```diff
@@ -0,0 +1,275 @@
+package proposals
+
+import (
+	"fmt"
+	"time"
+
+	"code.vegaprotocol.io/vega/libs/ptr"
+	"code.vegaprotocol.io/vega/protos/vega"
+	commandspb "code.vegaprotocol.io/vega/protos/vega/commands/v1"
+	datav1 "code.vegaprotocol.io/vega/protos/vega/data/v1"
+	"github.com/vegaprotocol/devopstools/tools"
+)
+
+func NewBTCUSDPerpetualMarketProposal(
+	settlementVegaAssetId string,
+	decimalPlaces uint64,
+	oraclePubKey string,
+	closingTime time.Time,
+	enactmentTime time.Time,
+	extraMetadata []string,
+) *commandspb.ProposalSubmission {
+	var (
+		reference = tools.RandAlpaNumericString(40)
+		Name      = fmt.Sprintf("BTCUSD Perpetual")
+	)
+
+	contractABI := "{\"inputs\":[],\"name\":\"latestAnswer\",\"outputs\":[{\"internalType\":\"int256\",\"name\":\"\",\"type\":\"int256\"}],\"stateMutability\":\"view\",\"type\":\"function\"}"
```

Comment:
I'm not 100% here, but this might need extra bracket like this, let's try it if it doesn't work at first

```suggestion
	contractABI := "[{\"inputs\":[],\"name\":\"latestAnswer\",\"outputs\":[{\"internalType\":\"int256\",\"name\":\"\",\"type\":\"int256\"}],\"stateMutability\":\"view\",\"type\":\"function\"}]"
```